test/cell_test.rb

```diff
@@ -52,32 +52,28 @@ def test_fire_upon_updates_fired_upon_to_true
   end
 
   def test_it_displays_a_dot_if_it_has_not_been_fired_upon
+    @cell.place_ship(@ship)
     assert_equal ".", @cell.render
   end
 
   def test_it_displays_an_m_if_it_has_been_fired_upon_with_no_ship
     @cell.fire_upon
-
     assert_equal "M", @cell.render
   end
 
   def test_it_displays_a_dot_after_placing_a_ship
     @cell.place_ship(@ship)
-
     assert_equal ".", @cell.render
   end
 
   def test_it_displays_an_s_if_it_has_a_ship_with_render_true_argument
     @cell.place_ship(@ship)
-    @cell.render(true)
-
-    assert_equal "S", @cell.render
+    assert_equal "S", @cell.render(true)
```

Comment:
This looks great! My brain wasn't working yesterday, and this makes a lot more sense.